.github/workflows/danger-pr-labels.yml

```diff
@@ -10,9 +10,32 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-      - name: Validate PR Labels
-        uses: danger/danger-js@9.1.8
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node_version: 10.x
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Cache Node Modules
+        id: cache-node-modules
+        uses: actions/cache@v1
         with:
-          args: "--dangerfile Automattic/peril-settings/org/pr/label.ts"
+          path: node_modules
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-node_modules
+
+      - name: Yarn Install
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        # frozen-lockfile will make the build fail if the lockfile is not there
+        run: yarn install --frozen-lockfile
+
+      - name: Validate PR Labels
```

Comment:
This differs from the job name which is "Validated PR Labels". IMHO saying "Validate" is better as it's a present-tense verb. Can we use it everywhere? :)

Reply:
Great suggestion. I do like the present-tense verb approach ❤️ . I'm onto it. 👍 .

Reply:
Oh :o. Just seen that past-tense "Validate_d_" in the code. Not intentional, just a typo. Good catch 😄